Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -1523,7 +1524,8 @@ - (void)disappearingTimerConfigurationViewWasTapped:(DisappearingTimerConfigurat
 
 - (void)autoLoadMoreIfNecessary
 {
-    if (self.isUserScrolling || !self.isViewVisible || self.isAppInBackground) {
+    BOOL isAppInBackground = CurrentAppContext().isInBackground;
```

Comment:
Do we want `!CurrentAppContext.isForegroundAndActive`?

Reply:
Better, done.